openedx_learning/core/contents/api.py

```diff
@@ -80,69 +40,122 @@ def get_or_create_media_type_id(mime_type: str) -> int:
         suffix = ""
 
     main_type, sub_type = base.split("/")
-    mt, _created = MediaType.objects.get_or_create(
+    media_type, _created = MediaType.objects.get_or_create(
         type=main_type,
         sub_type=sub_type,
         suffix=suffix,
     )
 
-    return mt.id
+    return media_type
+
+
+def get_content(content_id: int, /) -> Content:
+    """
+    Get a single Content object by its ID.
+
+    Content is always attached to something when it's created, like to a
+    ComponentVersion. That means the "right" way to access a Content is almost
+    always going to be via those relations and not via this function. But I
+    include this function anyway because it's tiny to write and it's better than
+    someone using a get_or_create_* function when they really just want to get.
+    """
+    return Content.objects.get(id=content_id)
 
 
-def get_or_create_raw_content(
+def get_or_create_text_content(
     learning_package_id: int,
     /,
-    data_bytes: bytes,
-    mime_type: str,
+    media_type_id: int,
```

Comment:
Consider putting `media_type_id` above the `/` to avoid committing to `_id` in the kwarg name.